2023-01-03 main branch e2e-failures CentOS 7 ARM64

A large number of failures are expected due to OCI mode limitations on CentOS 7, and inability to use buildkit on a distro this old.

These need to be investigated and gated appropriately.

```
--- FAIL: TestE2E (601.15s)
    --- FAIL: TestE2E/PAR (0.00s)
        --- FAIL: TestE2E/PAR/PULL (0.00s)
            --- FAIL: TestE2E/PAR/PULL/ordered (7.36s)
        --- FAIL: TestE2E/PAR/PUSH (0.00s)
            --- FAIL: TestE2E/PAR/PUSH/oras (1.12s)
                --- FAIL: TestE2E/PAR/PUSH/oras/oras_OCI-SIF_push (0.01s)
                --- FAIL: TestE2E/PAR/PUSH/oras/docker_OCI-SIF_push (0.02s)
        --- FAIL: TestE2E/PAR/VERIFY (0.00s)
            --- FAIL: TestE2E/PAR/VERIFY/ordered (0.24s)
                --- FAIL: TestE2E/PAR/VERIFY/ordered/Verify (0.22s)
                    --- FAIL: TestE2E/PAR/VERIFY/ordered/Verify/startOCSPResponder (0.01s)
        --- FAIL: TestE2E/PAR/SECURITY (0.00s)
            --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities (1.80s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/DefaultUser/OCIUser (0.08s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/DefaultRoot/OCIRoot (0.08s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/DefaultRoot/OCIFakeroot (0.09s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/NoPrivs/OCIRoot (0.08s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/NoPrivs/OCIFakeroot (0.10s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/NoPrivs/OCIUser (0.11s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/KeepPrivsUser/OCIUser (0.07s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/KeepPrivsRoot/OCIRoot (0.07s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/KeepPrivsRoot/OCIFakeroot (0.07s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/DropChownUser/OCIUser (0.11s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/DropChownRoot/OCIRoot (0.08s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/DropChownRoot/OCIFakeroot (0.08s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/AddSysAdminUser/OCIUser (0.10s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/AddSysAdminRoot/OCIRoot (0.10s)
                --- FAIL: TestE2E/PAR/SECURITY/ociCapabilities/AddSysAdminRoot/OCIFakeroot (0.10s)
        --- FAIL: TestE2E/PAR/SINGULARITYENV (0.00s)
            --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_singularityenv (23.84s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_singularityenv/DefaultPath (0.78s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_singularityenv/AppendToDefaultPath (0.65s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_singularityenv/PrependToDefaultPath (0.40s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_singularityenv/OverwriteDefaultPath (0.15s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_singularityenv/OverwriteTrailingCommaPath (0.17s)
            --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option (21.80s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/DefaultPath (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/DefaultPath#01 (0.14s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/DefaultPathOverride (0.15s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/DefaultPathOverride#01 (0.13s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/AppendDefaultPath (0.50s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/AppendDefaultPath#01 (0.13s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/PrependDefaultPath (0.13s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/PrependDefaultPath#01 (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestMultiLine (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestMultiLine#01 (0.09s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestEscapedNewline (0.09s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestEscapedNewline#01 (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestInvalidKey (0.11s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestInvalidKey#01 (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestDefaultLdLibraryPath (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestDefaultLdLibraryPath#01 (0.13s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestCustomTrailingCommaPath (0.08s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestCustomTrailingCommaPath#01 (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestCustomLdLibraryPath (0.11s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/TestCustomLdLibraryPath#01 (0.14s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_option/SINGULARITY_NAME_OCI_SIF (0.10s)
            --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file (8.02s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/DefaultPathOverride (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/DefaultPathOverrideEnvOptionPrecedence (0.11s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/DefaultPathOverrideEnvOptionPrecedence#02 (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/AppendDefaultPath (0.07s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/PrependDefaultPath (0.07s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/DefaultLdLibraryPath (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/CustomLdLibraryPath (0.09s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/CustomTrailingCommaPath (0.10s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/HostEnvUnset (0.09s)
                --- FAIL: TestE2E/PAR/SINGULARITYENV/oci_environment_file/HostEnvSet (0.05s)
        --- FAIL: TestE2E/PAR/DOCKER (0.00s)
            --- FAIL: TestE2E/PAR/DOCKER/docker_host (59.39s)
                --- FAIL: TestE2E/PAR/DOCKER/docker_host/OCIRoot (24.25s)
                    --- FAIL: TestE2E/PAR/DOCKER/docker_host/OCIRoot/pull (3.52s)
                        --- FAIL: TestE2E/PAR/DOCKER/docker_host/OCIRoot/pull/singularityDockerHostEmpty (0.86s)
                        --- FAIL: TestE2E/PAR/DOCKER/docker_host/OCIRoot/pull/dockerHostEmpty (0.85s)
                        --- FAIL: TestE2E/PAR/DOCKER/docker_host/OCIRoot/pull/singularityDockerHostValid (0.83s)
                        --- FAIL: TestE2E/PAR/DOCKER/docker_host/OCIRoot/pull/dockerHostValid (0.85s)
            --- FAIL: TestE2E/PAR/DOCKER/ordered (176.49s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/AUFS (5.79s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/AUFS/OCISIF (1.48s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/AUFS/OCISIFKeepLayers (1.46s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/user (15.32s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_df_build (2.71s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_df_build (2.19s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_oci-sif (0.76s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_oci-sif/OCIUser (0.08s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_oci-sif/OCIFakeroot (0.08s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_oci-sif/OCIRoot (0.08s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_oci-sif/OrigHomeOCIUser (0.07s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_oci-sif/OrigHomeOCIFakeroot (0.09s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/root_oci-sif/OrigHomeOCIRoot (0.09s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/user/uri (7.40s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/uri/OCIUser (1.13s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/uri/OCIFakeroot (0.82s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/uri/OCIRoot (0.98s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/uri/OrigHomeOCIUser (0.77s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/uri/OrigHomeOCIFakeroot (0.76s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/uri/OrigHomeOCIRoot (0.72s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_oci-sif (0.81s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_oci-sif/OCIUser (0.08s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_oci-sif/OCIFakeroot (0.08s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_oci-sif/OCIRoot (0.06s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_oci-sif/OrigHomeOCIUser (0.08s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_oci-sif/OrigHomeOCIFakeroot (0.08s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/user/user_oci-sif/OrigHomeOCIRoot (0.08s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/platform (5.22s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/platform/OCIUser/MultiArchArm64 (0.66s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/platform/OCIUser/MultiArchPpc64le (0.65s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/platform/OCIUser/SingleArchArm64 (0.63s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/platform/OCIUser/SingleArchPpc64le (0.85s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/crossarch_buildkit (1.93s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/crossarch_buildkit/User (1.93s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/scif (38.00s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/scif/build (37.45s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/scif/run_echo (0.10s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/scif/exec_echo (0.12s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/scif/run_script (0.12s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/scif/exec_script (0.08s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/scif/exec_script2 (0.13s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1286 (25.63s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1286/OCIFakeroot (1.25s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1286/OCIUser (0.77s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1286/OCIRoot (0.44s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528 (1.66s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIRoot (0.09s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIRoot/issue1528 (0.09s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIFakeroot (0.19s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIFakeroot/issue1528 (0.19s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIUser (0.10s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIUser/issue1528 (0.10s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIUser#01 (0.09s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIUser#01/issue1528override (0.09s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIRoot#01 (0.12s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIRoot#01/issue1528override (0.12s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIFakeroot#01 (0.30s)
                        --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1528/OCIFakeroot#01/issue1528override (0.30s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1586 (4.84s)
                --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1670 (2.62s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1670/OCIUser (0.81s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1670/OCIRoot (0.69s)
                    --- FAIL: TestE2E/PAR/DOCKER/ordered/issue_1670/OCIFakeroot (1.11s)
        --- FAIL: TestE2E/PAR/ACTIONS (0.01s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals (1.73s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit0/OCIUser (0.07s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit1/OCIUser (0.09s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit134/OCIUser (0.09s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/SignalKill/OCIUser (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/SignalAbort/OCIUser (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit0/OCIRoot (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit1/OCIRoot (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit134/OCIRoot (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/SignalKill/OCIRoot (0.06s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/SignalAbort/OCIRoot (0.07s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit0/OCIFakeroot (0.06s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit1/OCIFakeroot (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/Exit134/OCIFakeroot (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/SignalKill/OCIFakeroot (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExitSignals/SignalAbort/OCIFakeroot (0.07s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociBindImage (1.50s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociBindImage/NoBindOption (0.06s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociBindImage/Squashfs (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociBindImage/SquashfsDouble (0.12s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociBindImage/SquashfsMixedBind (0.07s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociBinds (6.22s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser (2.34s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/SimpleFile (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/SimpleDir (0.33s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/HomeOverride (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/TmpOverride (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/VarTmpOverride (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/NestedBindFile (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/NestedBindDir (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/MultipleNestedBindDir (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/WorkdirTmpBind (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/WorkdirVarTmpBind (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/WorkdirVarTmpBindWritable (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/IsScratchTmpfs (0.13s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/BindOverScratch (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/ScratchTmpfsBind (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/ScratchWorkdirBind (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/CustomHomeOneToOne (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/CustomHomeBind (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/MountSingle (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIUser/MountNested (0.12s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot (1.60s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/SimpleFile (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/SimpleDir (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/HomeOverride (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/TmpOverride (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/VarTmpOverride (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/NestedBindFile (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/NestedBindDir (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/MultipleNestedBindDir (0.05s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/WorkdirTmpBind (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/WorkdirVarTmpBind (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/WorkdirVarTmpBindWritable (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/IsScratchTmpfs (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/BindOverScratch (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/ScratchTmpfsBind (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/ScratchWorkdirBind (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/CustomHomeOneToOne (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/CustomHomeBind (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/MountSingle (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIRoot/MountNested (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot (1.79s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/SimpleFile (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/SimpleDir (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/HomeOverride (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/TmpOverride (0.12s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/VarTmpOverride (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/NestedBindFile (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/NestedBindDir (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/MultipleNestedBindDir (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/WorkdirTmpBind (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/WorkdirVarTmpBind (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/WorkdirVarTmpBindWritable (0.14s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/IsScratchTmpfs (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/BindOverScratch (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/ScratchTmpfsBind (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/ScratchWorkdirBind (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/CustomHomeOneToOne (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/CustomHomeBind (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/MountSingle (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociBinds/OCIFakeroot/MountNested (0.10s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociHomeCwdPasswd (1.01s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociHomeCwdPasswd/OCIUser/cwd (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociHomeCwdPasswd/OCIUser/passwd (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociHomeCwdPasswd/OCIRoot/cwd (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociHomeCwdPasswd/OCIRoot/passwd (0.11s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociHomeCwdPasswd/OCIFakeroot/cwd (0.07s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociHomeCwdPasswd/OCIFakeroot/passwd (0.10s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount (1.72s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount/proc (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount/sys (0.11s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount/tmp (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount/home (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount/cwd (0.11s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount//etc/hosts (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount//etc/localtime (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount/binds-paths-hosts (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNo-mount/binds-paths-localtime (0.11s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociAllowSetuid (0.67s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociAllowSetuid/False (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociAllowSetuid/True (0.09s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociRun (29.43s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser (9.49s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser/oci-sif (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser/oci-sif_bare (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser/oci-sif-oras (0.14s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser/docker-archive (0.84s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser/oci-archive (0.88s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser/oci (0.87s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIUser/true (1.09s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot (9.98s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot/oci-sif (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot/oci-sif_bare (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot/oci-sif-oras (0.13s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot/docker-archive (0.87s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot/oci-archive (1.02s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot/oci (0.87s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIRoot/true (1.04s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot (8.78s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot/oci-sif (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot/oci-sif_bare (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot/oci-sif-oras (0.15s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot/docker-archive (0.83s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot/oci-archive (0.79s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot/oci (0.78s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociRun/OCIFakeroot/true (1.13s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociSTDPIPE (0.94s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociSTDPIPE/TrueSTDIN (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociSTDPIPE/FalseSTDIN (0.09s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociSTDPIPE/CwdPath (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociSTDPIPE/PwdPath (0.09s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociSTDPIPE/id (0.10s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociIDMaps (0.68s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociIDMaps/OCIUser (0.13s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociIDMaps/OCIUser/OCIUser (0.13s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociIDMaps/OCIFakeroot (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociIDMaps/OCIFakeroot/OCIFakeroot (0.07s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociCdi (0.99s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIFakeroot (0.20s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIFakeroot/ValidMounts (0.12s)
                        --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIFakeroot/ValidMounts/ValidMounts (0.12s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIUser (0.16s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIUser/ValidMounts (0.08s)
                        --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIUser/ValidMounts/ValidMounts (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIRoot (0.14s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIRoot/ValidMounts (0.07s)
                        --- FAIL: TestE2E/PAR/ACTIONS/ociCdi/OCIRoot/ValidMounts/ValidMounts (0.07s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay (3.71s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIFakeroot (1.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIFakeroot/ExistRWDir (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIFakeroot/ExistRWDirRevisit (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIFakeroot/ExistRWDirRevisitAsRO (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIFakeroot/AutoAddTmpfs (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIFakeroot/SeveralRODirs (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIFakeroot/SquashfsAndDirs (0.09s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIUser (1.16s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIUser/ExistRWDir (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIUser/ExistRWDirRevisit (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIUser/ExistRWDirRevisitAsRO (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIUser/AutoAddTmpfs (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIUser/SeveralRODirs (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIUser/SquashfsAndDirs (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIRoot (0.94s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIRoot/ExistRWDir (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIRoot/ExistRWDirRevisit (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIRoot/ExistRWDirRevisitAsRO (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIRoot/AutoAddTmpfs (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIRoot/SeveralRODirs (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociOverlay/OCIRoot/SquashfsAndDirs (0.09s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociNetwork (0.95s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNetwork/NoneNetworkRoot (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNetwork/NoneNetworkUser (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociNetwork/NoneNetworkFakeRoot (0.07s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociExec (6.39s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser (2.14s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/True (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/TrueAbsPAth (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/False (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/FalseAbsPath (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/TouchTmp (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/TouchVarTmp (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/TouchHome (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/Home (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/HomePath (0.12s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/HomeTmp (0.12s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/HomeTmpExplicit (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/NoHome (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/UTSNamespace (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/Hostname (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/Workdir (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/Pwd (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/Cwd (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/ResolvConfGoogle (0.12s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/ResolvConfCloudflare (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/CustomHomePreservesRootShell (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/Containlibs (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/DefaultPID (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIUser/NoPID (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot (1.69s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/True (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/TrueAbsPAth (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/False (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/FalseAbsPath (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/TouchTmp (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/TouchVarTmp (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/TouchHome (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/Home (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/HomePath (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/HomeTmp (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/HomeTmpExplicit (0.05s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/NoHome (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/UTSNamespace (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/Hostname (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/Workdir (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/Pwd (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/Cwd (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/ResolvConfGoogle (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/ResolvConfCloudflare (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/CustomHomePreservesRootShell (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/Containlibs (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/DefaultPID (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIRoot/NoPID (0.07s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot (2.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/True (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/TrueAbsPAth (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/False (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/FalseAbsPath (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/TouchTmp (0.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/TouchVarTmp (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/TouchHome (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/Home (0.06s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/HomePath (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/HomeTmp (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/HomeTmpExplicit (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/NoHome (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/UTSNamespace (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/Hostname (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/Workdir (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/Pwd (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/Cwd (0.08s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/ResolvConfGoogle (0.16s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/ResolvConfCloudflare (0.07s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/CustomHomePreservesRootShell (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/Containlibs (0.09s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/DefaultPID (0.10s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociExec/OCIFakeroot/NoPID (0.10s)
            --- FAIL: TestE2E/PAR/ACTIONS/ociShell (31.33s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIUser (10.15s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIUser/ShellExit (10.05s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIUser/ShellBadCommand (0.10s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIRoot (10.11s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIRoot/ShellExit (10.03s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIRoot/ShellBadCommand (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIFakeroot (10.14s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIFakeroot/ShellExit (10.04s)
                    --- FAIL: TestE2E/PAR/ACTIONS/ociShell/OCIFakeroot/ShellBadCommand (0.10s)
    --- FAIL: TestE2E/SEQ (275.07s)
        --- FAIL: TestE2E/SEQ/REGISTRY (4.37s)
            --- FAIL: TestE2E/SEQ/REGISTRY/auth (3.74s)
                --- FAIL: TestE2E/SEQ/REGISTRY/auth/default (1.88s)
                    --- FAIL: TestE2E/SEQ/REGISTRY/auth/default/docker_pull_ocisif (0.49s)
                    --- FAIL: TestE2E/SEQ/REGISTRY/auth/default/docker_push (0.01s)
                --- FAIL: TestE2E/SEQ/REGISTRY/auth/custom (1.85s)
                    --- FAIL: TestE2E/SEQ/REGISTRY/auth/custom/docker_pull_ocisif (0.46s)
                    --- FAIL: TestE2E/SEQ/REGISTRY/auth/custom/docker_push (0.01s)
        --- FAIL: TestE2E/SEQ/DOCKER (111.15s)
            --- FAIL: TestE2E/SEQ/DOCKER/cred_prio (111.15s)
                --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User (55.81s)
                    --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/def_df_build (17.40s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/def_df_build/pfx_denv_right,_no_auth (1.65s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/def_df_build/pfx_denv_right,_auth (1.71s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/def_df_build/nopfx_denv_right,_no_auth (1.61s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/def_df_build/nopfx_denv_right,_auth (1.66s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/def_df_build/both_denv_(pfx_right),_auth (1.63s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/def_df_build/both_denv_(pfx_right),_noauth (1.57s)
                    --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/cstm_df_build (17.21s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/cstm_df_build/pfx_denv_right,_no_auth (1.56s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/cstm_df_build/pfx_denv_right,_auth (1.60s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/cstm_df_build/nopfx_denv_right,_no_auth (1.62s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/cstm_df_build/nopfx_denv_right,_auth (1.64s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/cstm_df_build/both_denv_(pfx_right),_auth (1.61s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/User/cstm_df_build/both_denv_(pfx_right),_noauth (1.62s)
                --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root (55.34s)
                    --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/def_df_build (17.29s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/def_df_build/pfx_denv_right,_no_auth (1.61s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/def_df_build/pfx_denv_right,_auth (1.62s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/def_df_build/nopfx_denv_right,_no_auth (1.60s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/def_df_build/nopfx_denv_right,_auth (1.69s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/def_df_build/both_denv_(pfx_right),_auth (1.62s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/def_df_build/both_denv_(pfx_right),_noauth (1.64s)
                    --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/cstm_df_build (17.12s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/cstm_df_build/pfx_denv_right,_no_auth (1.61s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/cstm_df_build/pfx_denv_right,_auth (1.63s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/cstm_df_build/nopfx_denv_right,_no_auth (1.60s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/cstm_df_build/nopfx_denv_right,_auth (1.61s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/cstm_df_build/both_denv_(pfx_right),_auth (1.59s)
                        --- FAIL: TestE2E/SEQ/DOCKER/cred_prio/Root/cstm_df_build/both_denv_(pfx_right),_noauth (1.59s)
        --- FAIL: TestE2E/SEQ/ACTIONS (32.51s)
            --- FAIL: TestE2E/SEQ/ACTIONS/FUSEImage (3.06s)
                --- FAIL: TestE2E/SEQ/ACTIONS/FUSEImage/User (0.60s)
                --- FAIL: TestE2E/SEQ/ACTIONS/FUSEImage/Root (0.64s)
                --- FAIL: TestE2E/SEQ/ACTIONS/FUSEImage/Fakeroot (0.58s)
                --- FAIL: TestE2E/SEQ/ACTIONS/FUSEImage/UserNamespace (0.63s)
                --- FAIL: TestE2E/SEQ/ACTIONS/FUSEImage/RootUserNamespace (0.60s)
            --- FAIL: TestE2E/SEQ/ACTIONS/auth (15.37s)
                --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIUser (3.91s)
                    --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIUser/default (2.03s)
                        --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIUser/default/docker (0.49s)
                    --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIUser/custom (1.88s)
                        --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIUser/custom/docker (0.48s)
                --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIRoot (4.01s)
                    --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIRoot/default (2.08s)
                        --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIRoot/default/docker (0.50s)
                    --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIRoot/custom (1.93s)
                        --- FAIL: TestE2E/SEQ/ACTIONS/auth/OCIRoot/custom/docker (0.49s)
            --- FAIL: TestE2E/SEQ/ACTIONS/ociRelWorkdirScratch (0.72s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociRelWorkdirScratch/OCIUser (0.09s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociRelWorkdirScratch/OCIRoot (0.09s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociRelWorkdirScratch/OCIFakeroot (0.08s)
            --- FAIL: TestE2E/SEQ/ACTIONS/ociCompat (0.79s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociCompat/containall (0.11s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociCompat/writable-tmpfs (0.08s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociCompat/no-init (0.09s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociCompat/no-umask (0.07s)
            --- FAIL: TestE2E/SEQ/ACTIONS/ociNoCompat (2.27s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociNoCompat/dirBinds (0.07s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociNoCompat/bindPath (0.09s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociNoCompat/cwd (0.08s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociNoCompat/cwdBind (0.09s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociNoCompat/readOnly (0.08s)
                --- FAIL: TestE2E/SEQ/ACTIONS/ociNoCompat/writableTmpfs (0.09s)
            --- FAIL: TestE2E/SEQ/ACTIONS/ociOverlayTeardown (0.57s)
        --- FAIL: TestE2E/SEQ/CONFIG (4.60s)
            --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global (2.91s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/ConfigPasswdNo (0.10s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/ConfigPasswdYes (0.13s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/ConfigGroupNo (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/ConfigGroupYes (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/ConfigResolvConfNo (0.10s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/ConfigResolvConfYes (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountProcNo (0.10s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountProcYes (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountSysNo (0.11s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountSysYes (0.11s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountDevMinimal (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountDevYes (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountDevPtsYes (0.11s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountHomeNo (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountHomeNoCorrectDir (0.12s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountHomeYes (0.11s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountTmpNo (0.10s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/MountTmpYes (0.10s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/UserBindControlNo (0.09s)
                --- FAIL: TestE2E/SEQ/CONFIG/oci_config_global/UserBindControlYes (0.12s)
        --- FAIL: TestE2E/SEQ/CGROUPS (33.00s)
            --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups (5.15s)
                --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/systemd (2.46s)
                    --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/systemd/OCIRoot (0.37s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/systemd/OCIRoot/memory_limit_oci (0.11s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/systemd/OCIRoot/cpu_success (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/systemd/OCIRoot/device_allow_default (0.06s)
                --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/cgroupfs (2.65s)
                    --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/cgroupfs/OCIRoot (0.41s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/cgroupfs/OCIRoot/memory_limit_oci (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/cgroupfs/OCIRoot/cpu_success (0.08s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_root_cgroups/cgroupfs/OCIRoot/device_allow_default (0.08s)
            --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups (8.76s)
                --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd (4.30s)
                    --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot (0.56s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/cpus (0.08s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/cpu-shares (0.06s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/cpuset-cpus (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/cpuset-mems (0.06s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/memory (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/memory-reservation (0.06s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/memory-swap (0.06s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/systemd/OCIRoot/oom-kill-disable (0.10s)
                --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs (4.42s)
                    --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot (0.58s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/cpus (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/cpu-shares (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/cpuset-cpus (0.06s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/cpuset-mems (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/memory (0.10s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/memory-reservation (0.07s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/memory-swap (0.06s)
                        --- FAIL: TestE2E/SEQ/CGROUPS/action_flags_root_cgroups/cgroupfs/OCIRoot/oom-kill-disable (0.08s)
        --- FAIL: TestE2E/SEQ/BUILD (46.69s)
            --- FAIL: TestE2E/SEQ/BUILD/dockerfile (43.07s)
                --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User (21.52s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/simple (1.66s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/simple/build (1.56s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/simple/act (0.10s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_none (2.40s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_none/build (2.33s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_none/act (0.07s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_wrong (1.67s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_wrong/build (1.60s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_wrong/act (0.07s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_wrong_and_right (2.31s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_wrong_and_right/build (2.24s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_wrong_and_right/act (0.07s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_right (1.68s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_right/build (1.58s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_right/act (0.10s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_none (2.24s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_none/build (2.14s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_none/act (0.10s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_wrong (1.70s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_wrong/build (1.62s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_wrong/act (0.08s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_wrong_and_right (1.67s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_wrong_and_right/build (1.59s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_wrong_and_right/act (0.08s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_right (1.69s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_right/build (1.60s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/ba_nd_right/act (0.09s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/add (1.94s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/add/build (1.85s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/User/add/act (0.09s)
                --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root (21.55s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/simple (1.63s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/simple/build (1.57s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/simple/act (0.06s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_none (1.89s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_none/build (1.82s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_none/act (0.07s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_wrong (1.90s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_wrong/build (1.83s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_wrong/act (0.07s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_wrong_and_right (1.94s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_wrong_and_right/build (1.87s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_wrong_and_right/act (0.07s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_right (1.89s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_right/build (1.81s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_right/act (0.08s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_none (1.91s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_none/build (1.83s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_none/act (0.08s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_wrong (1.94s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_wrong/build (1.88s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_wrong/act (0.06s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_wrong_and_right (2.03s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_wrong_and_right/build (1.96s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_wrong_and_right/act (0.07s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_right (1.94s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_right/build (1.86s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/ba_nd_right/act (0.08s)
                    --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/add (1.90s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/add/build (1.83s)
                        --- FAIL: TestE2E/SEQ/BUILD/dockerfile/Root/add/act (0.07s)
```

--- a/e2e/internal/e2e/profile.go
+++ b/e2e/internal/e2e/profile.go
@@ -277,7 +277,7 @@ func fakerootRequirements(t *testing.T) {
 func ociRequirements(t *testing.T) {
 	require.Kernel(t, 4, 18) // FUSE in userns
 	require.UserNamespace(t)
-	require.Command(t, "runc")
+	require.OneCommand(t, []string{"runc", "crun"})
 	require.OneCommand(t, []string{"sqfstar", "tar2sqfs"})
 
 	uid := uint32(origUID)

--- a/e2e/internal/e2e/profile.go
+++ b/e2e/internal/e2e/profile.go
@@ -278,6 +278,7 @@ func ociRequirements(t *testing.T) {
 	require.Kernel(t, 4, 18) // FUSE in userns
 	require.UserNamespace(t)
 	require.Command(t, "runc")
+	require.OneCommand(t, []string{"sqfstar", "tar2sqfs"})
 
 	uid := uint32(origUID)
 

--- a/internal/pkg/test/tool/require/require.go
+++ b/internal/pkg/test/tool/require/require.go
@@ -270,6 +270,23 @@ func Command(t *testing.T, command string) {
 	t.Skipf("%s command not found in $PATH", command)
 }
 
+// OneCommand checks if one of the provided commands is available (via
+// Singularity's internal bin.FindBin() facility, or else simply on the PATH).
+// If none are found, the current test is skipped with a message.
+func OneCommand(t *testing.T, commands []string) {
+	for _, c := range commands {
+		if _, err := bin.FindBin(c); err == nil {
+			return
+		}
+
+		if _, err := exec.LookPath(c); err == nil {
+			return
+		}
+	}
+
+	t.Skipf("%v commands not found in $PATH", commands)
+}
+
 // Seccomp checks that seccomp is enabled, if not the
 // current test is skipped with a message.
 func Seccomp(t *testing.T) {

--- a/internal/pkg/util/bin/bin.go
+++ b/internal/pkg/util/bin/bin.go
@@ -53,6 +53,9 @@ func FindBin(name string) (path string, err error) {
 	// unprivileged overlays
 	case "fuse-overlayfs":
 		return findOnPath(name)
+	// tar to squashfs tools for OCI-mode image conversion
+	case "tar2sqfs", "sqfstar":
+		return findOnPath(name)
 	default:
 		return "", fmt.Errorf("executable name %q is not known to FindBin", name)
 	}

